engine/runtime/src/bench/java/org/enso/interpreter/bench/benchmarks/semantic/TypePatternBenchmarks.java

```diff
@@ -88,7 +88,7 @@ public void matchOverAny(Blackhole matter) {
   }
 
   @Benchmark
-  public void matchOverDecimal(Blackhole matter) {
+  public void matchOverFloat(Blackhole matter) {
```

Comment:
Changing name of a `@Benchmark` will disconnect it from the history of previous measurements and [affect the chart](https://enso-org.github.io/engine-benchmark-results/engine-benchs.html).

Reply:
Reverted with a comment

Reply:
Another use-case for *versioning* mentioned in https://github.com/enso-org/enso/issues/5714.